### PR TITLE
Fixed taxonomy query var might be NULL.

### PR DESCRIPTION
**Pull request by fabianmarz**
### Description
- Regression fix for #28

### CODE REVIEW

File: src/ProductFilters/WidgetFilterDeliveryTime.php [hunk 1]
```diff
@@ -154,8 +154,8 @@ public static function getProductsDeliveryTimes(array $args = []) {
    *
    * @implements get_terms_orderby
    */
-  public static function get_terms_orderby(string $orderby, array $query_vars, array $taxonomy): string {
-    if (in_array('product_delivery_times', $taxonomy, TRUE) && $query_vars['orderby'] === 'slug') {
+  public static function get_terms_orderby(string $orderby, array $query_vars, ?array $taxonomy): string {
```
Comment (sun) [inline src/ProductFilters/WidgetFilterDeliveryTime.php]: Wasn't aware that optional function parameters were introduced already — and actually can't find it in the PHP manual.  Do you have a link?

No mention of this no any of these:
https://www.php.net/manual/en/migration70.new-features.php
https://www.php.net/manual/en/migration71.new-features.php
https://www.php.net/manual/en/migration72.new-features.php
https://www.php.net/manual/en/migration73.new-features.php

Comment (sun) [reply src/ProductFilters/WidgetFilterDeliveryTime.php]:
```suggestion
  public static function get_terms_orderby(string $orderby, array $query_vars, array $taxonomy = []): string {
```

Comment (juanlopez4691) [reply src/ProductFilters/WidgetFilterDeliveryTime.php]: @sun It's the first "new feature" for PHP 7.1 https://www.php.net/manual/en/migration71.new-features.php

Comment (fabianmarz) [reply src/ProductFilters/WidgetFilterDeliveryTime.php]: As stated by @juanlopez4691, to cite the docs:
> Type declarations for parameters and return values can now be marked as nullable by prefixing the type name with a question mark.

**Pull request merged**